src/renderer/pages/Settings/Networks/components/NetworkList/NetworkList.tsx

```diff
@@ -1,42 +1,54 @@
 import { useEffect, useState, ReactNode } from 'react';
+import { useUnit } from 'effector-react';
 
 import { ExtendedChain, networkUtils } from '@entities/network';
 import { CaptionText, Counter, Accordion } from '@shared/ui';
+import { filterModel } from '@features/networks/NetworkFilter/model/network-filter';
 
 type Props = {
   title: string;
   isDefaultOpen?: boolean;
-  query?: string;
   networkList: ExtendedChain[];
   children: (network: ExtendedChain) => ReactNode;
 };
 
-export const NetworkList = ({ title, isDefaultOpen, query, networkList, children }: Props) => {
+export const NetworkList = ({ title, isDefaultOpen, networkList, children }: Props) => {
   const [isListOpen, setIsListOpen] = useState(isDefaultOpen);
+  const filterQuery = useUnit(filterModel.$filterQuery);
+
+  console.log('filterQuery', filterQuery);
+  console.log('isDefaultOpen', isDefaultOpen);
+  console.log('isListOpen', isListOpen);
 
   useEffect(() => {
-    if (query) {
-      setIsListOpen(Boolean(query));
+    if (filterQuery) {
+      setIsListOpen(true);
     } else {
       setIsListOpen(isDefaultOpen);
     }
-  }, [query]);
+  }, [filterQuery]);
 
   if (networkList.length === 0) return null;
 
   const { success, connecting, error } = networkList.reduce(
```

Comment:
Should be moved inside model

Reply:
I moved it inside the `network-list-utils`, because it's linked to a specific network list (either active or inactive) and I think it's probably an even better place. Let me know if you think otherwise.